.travis.yml

```diff
@@ -60,8 +60,7 @@ addons:
     - rsyslog
 
 install: "travis_retry pip install tox coveralls"
-before_script: '[ "xxx$BOULDER_INTEGRATION" = "xxx" ] || ./tests/boulder-start.sh'
-script: 'travis_retry tox && ([ "xxx$BOULDER_INTEGRATION" = "xxx" ] || (source .tox/$TOXENV/bin/activate && ./tests/boulder-integration.sh))'
+script: 'travis_retry tox && ([ "xxx$BOULDER_INTEGRATION" = "xxx" ] || ./tests/travis-integration.sh)'
```

Comment:
They look pretty comparable to me! https://travis-ci.org/letsencrypt/letsencrypt/builds/91021463 (this branch) vs https://travis-ci.org/letsencrypt/letsencrypt/builds/90990967 (master)